test/utils/SchemaTester.js

```diff
@@ -0,0 +1,108 @@
+const testSchema = (Model) => ({
+    fieldRequired: (fieldName, customObject) => {
```

Comment:
```suggestion
    fieldRequired: (fieldName, customObject = {}) => {
```

Instead of using `|| {}` below, I think default argument values are clearer

Furthermore, `customObject` can be renamed to `instanceValues` or something similar, I think. I'm not sure of the best name for this variable, but I thin that this is clearer than `customObject`.